app/main.py

```diff
@@ -1,3 +1,79 @@
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: None) -> None:
```

Comment:
`other` can be int, float, or Distance. Improve the type hint for `other` in other methods too, please.

Reply:
Not fixed